src/ceph-volume/ceph_volume/util/system.py

```diff
@@ -273,3 +273,12 @@ def get_mounts(devices=False, paths=False, realpath=False):
         return devices_mounted
     else:
         return paths_mounted
+
+
+def set_context(path, recursive = False):
+    # restore selinux context to default policy values
+    if which('restorecon'):
```

Comment:
`which()` is not a boolean function, but this line is treating it like one. It is really trying to make an absolute path, and failing to do so, it will just return the executable (`restorecon` in this case). So this will always be a truthy value